doc/data/messages/i/invalid-unary-operand-type/bad.py

```diff
@@ -0,0 +1,2 @@
+fruit = int
+worm = - fruit  # [invalid-unary-operand-type]
```

Comment:
```suggestion
cherries = 10
eaten_cherries = int
cherries = - eaten_cherries  # [invalid-unary-operand-type]
```

Reply:
Yes, both examples make much more sense. :) Thank you very much.